src/components/artwork/fillwidth.tsx

```diff
@@ -127,8 +126,13 @@ const ArtworkFragment = Relay.QL`
     ${Artwork.getFragment("artwork")}
   }
 `
+const sizeMeOptions = {
```

Comment:
Listens to changes in size for the wrapped component and passes the dimensions down as props.